src/tasks/netcore/NetCoreTaskHelper.ts

```diff
@@ -154,19 +146,15 @@ export class NetCoreTaskHelper implements TaskHelper {
         runOptions.os = runOptions.os || 'Linux';
         runOptions.image = inferImageName(runDefinition as DockerRunTaskDefinition, context, context.folder.name, 'dev');
 
-        const ssl = helperOptions.configureSsl !== undefined ? helperOptions.configureSsl : await NetCoreTaskHelper.inferSsl(context.folder, helperOptions);
+        const ssl = !!helperOptions.configureSsl; // SSL will be enabled only if helperOptions.configureSsl is explicitly true
+        context.actionContext.telemetry.properties.netCoreSslSetting = helperOptions.configureSsl === undefined ? 'undefined' : helperOptions.configureSsl.toString();

```

Comment:
Just curious: we prefer to see "undefined" in telemetry, rather than empty string, or "(not set)" or something?

Reply:
I do prefer an explicit value vs. blank, as blank could imply that the telemetry pipeline dropped it, or it's an old version that doesn't set it, etc. As for "undefined" vs "unset" it doesn't matter much, but JS uses "undefined" all over and we're using "true" and "false" for explicitly-set values, so it seems good enough.